plugins/events/create.js

```diff
@@ -200,7 +200,9 @@ module.exports = () => ({
                 // User has good permissions, create an event
                 sha = await scm.getCommitSha(scmConfig);
             } catch (err) {
-                if (err.statusCode) {
+                if (err.statusCode && err.statusCode === 404) {
+                    throw boom.notFound(err.message);
+                } else {
```

Comment:
Why do you need this change? If err.statusCode contains 404, boomify seems to treat it as 404.

Reply:
This error has occured only under my private circumstance. So, I deleted.